openhexa/cli/cli.py

```diff
@@ -308,6 +309,56 @@ def pipelines_push(path: str):
             )
 
 
+@pipelines.command("delete")
+@click.argument("code", type=str)
+def pipelines_delete(code: str):
+    """
+    Delete a pipeline and all his versions.
+    """
+
+    user_config = open_config()
+    try:
+        workspace = user_config["openhexa"]["current_workspace"]
+    except KeyError:
+        click.echo(
+            "No workspace activated. Use openhexa workspaces add or openhexa workspaces activate to "
+            "activate a workspace.",
+            err=True,
+        )
+        sys.exit(1)
+    else:
+        pipeline = get_pipeline(user_config, code)
+        if get_pipeline(user_config, code) is None:
+            click.echo(
+                f"Pipeline {click.style(code, bold=True)} does not exist in workspace {click.style(workspace, bold=True)}"
+            )
+            sys.exit(1)
+
+        confirmation_code = click.prompt(
+            f"Removing {code} from workspace {workspace}. Please type {code} to confirm",
```

Comment:
Minor change:

`f'This will remove the pipeline "{code}" from the "{workspace} workspace. This operation cannot be undone.\nPlease enter "{code}" to confirm:'

Reply:
and let's use `click.style` like a couple of lines above